src/app.module.ts

```diff
@@ -6,6 +6,7 @@ import { Contributor, ContributorsModule } from '@/contributors';
 import { GithubModule } from '@/github';
 import { SynchronizationModule } from '@/synchronization';
 import { CommandsModule } from '@/commands';
+import { OnlydustModule } from './onlydust/onlydust.module';
```

Comment:
```suggestion
import { OnlydustModule } from './onlydust';
```

should be imported from a module index file